### PR TITLE
MD-enable python binding

### PR DESCRIPTION
**Pull request by vaidy4github**

### CODE REVIEW

**Comment by vaidy4github** [comment]
cc @drio

**Comment by drio** [comment]
Thanks Vaidy. I'll :eyes: shortly.

File: README.md [hunk 1]
```diff
@@ -186,6 +186,80 @@ for metric in api.list_metrics(name=" "):
...
+
+```
+  api = librato.connect(user, token, tags={'hostname': 'web-host-1'})
+  api.add_tags({'company': 'Librato'})        # Top-level tag set now has 'company' and 'hostname'
+  api.set_tags({'hostname': 'web-host-2'})    # Top-level tag set only has 'hostname'
+```
+
+### Querying tagged measurements
+Tagged measurements can be queried using the Connection `get_tagged()`. This method is much like `get()` but accepts a
+`tags_search` parameter, that specifies a tag query string. Refer to the main developer documentation for additional details
```
Comment (drio) [inline README.md]: We can link here to the knowledge base for easy access to the API details.

File: librato/exceptions.py [hunk 2]
```diff
@@ -78,9 +78,13 @@ def _parse_error_message(self):
             # The API could return 'errors' or just 'error' with a flat message
             if 'error' in self.error_payload:
                 return self.error_payload['error']
+            elif 'message' in self.error_payload:
+                return self.error_payload['message']
```
Comment (drio) [inline librato/exceptions.py]: :eyes:

**Comment by drio** [comment]
@vaidy4github Excellent work. :+1:

@josephruscio Once we merge this, can we push a new release or should we wait until we unleash the MD officially ?

**Comment by vaidy4github** [comment]
Thanks for reviewing, @drio!

File: README.md (hunk 1 above)
Comment (vaidy4github) [inline README.md]: Agreed. I believe the MD API docs are not officially out there yet. I am making a note to update the README once they get published.

**Pull request merged**